Parser/Parser.fs

```diff
@@ -283,8 +281,32 @@ module Parse =
             str_ws1 "package" >>. pFullIdent_ws .>> eostm
             |>> TPackage
 
+        /// Parser for map values in constants
+        let internal pConstant, internal pOptionDefR = createParserForwardedToRef<PConstant,State>()
+
+        let internal pColon = pstring ":" .>> ws
+        let internal pKey = manySatisfy (fun x -> isAsciiLetter x || x = '_') .>> ws
```

Comment:
`pKey` can be deleted; should use `pIdent` or `pIdent_ws` instead (the latter two both parse a non-dotted identifier).